### PR TITLE
Fix security issue found by sonar

### PR DESCRIPTION
**Pull request by kangkaisen**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] others

We shouldn‘t print password in log.

Mainly want to remove the following codes:

```
        } catch (SQLException e) {
            LOG.warn("fail to get connection to mysql. url={}, user={}, passwd={}, exception={}",
                    url, user, passwd, e.getMessage());
            return null;
```

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/load/BrokerLoadErrorHub.java [hunk 1]
```diff
@@ -105,18 +103,6 @@ public String getBrief() {
         }
     }
 
-    // Broker load error hub does not support showing detail error msg in 'show load warnings' stmt.
-    // User need to file error file in remote storage with file name showed in 'show load' stmt
-    @Override
-    public List<ErrorMsg> fetchLoadError(long jobId) {
```
Comment (trueeyu) [inline fe/fe-core/src/main/java/com/starrocks/load/BrokerLoadErrorHub.java:111]: fetchLoadError was not called

Comment (kangkaisen) [reply fe/fe-core/src/main/java/com/starrocks/load/BrokerLoadErrorHub.java:111]: yes

**Comment by kangkaisen** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by satanson — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/load/MysqlLoadErrorHub.java [hunk 2]
```diff
@@ -35,11 +33,6 @@
...
 import java.io.DataOutput;
 import java.io.IOException;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.List;
 import java.util.Map;
 
 public class MysqlLoadErrorHub extends LoadErrorHub {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/load/MysqlLoadErrorHub.java:38]: this file can be deleted

**Review by trueeyu — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/load/MysqlLoadErrorHub.java (hunk 2 above)
Comment (kangkaisen) [reply fe/fe-core/src/main/java/com/starrocks/load/MysqlLoadErrorHub.java:38]: OK, I will remove it in next PR

**Pull request merged**